frontend/lib/onboarding/relief-attempts.tsx

```diff
@@ -7,17 +7,18 @@ import { YesNoRadiosFormField } from "../forms/yes-no-radios-form-field";
 import { ProgressButtons } from "../ui/buttons";
 import { toStringifiedNullBool } from "../forms/form-input-converter";
 import { MiddleProgressStep } from "../progress/progress-step-route";
+import { Trans, t } from "@lingui/macro";
+import { li18n } from "../i18n-lingui";
 
 const ReliefAttemptsPage = MiddleProgressStep((props) => (
-  <Page title="Previous attempts to get help">
+  <Page title={li18n._(t`Previous attempts to get official help`)} withHeading>
     <div>
-      <h1 className="title is-4 is-spaced">
-        Previous attempts to get official help
-      </h1>
       <p className="subtitle is-6">
-        It is encouraged that you call 311 to report housing complaints directly
-        with the City. By calling, you will trigger a formal inspection process
-        that may lead to official violations being issued.
+        <Trans id="justfix.LocReliefAttemptsBlurb">
```

Comment:
Oof, I'm just noticing now that this whole file is in the `onboarding` directory but should probably be moved to the `loc` directory! Probably has to do w/ this question formerly being onboarding step 2...

Reply:
Good catch! Just moved it